incident/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
```

Comment:
For good practices, write at least one passing test for your app... Just for the experience